vc_zoom/indico_vc_zoom/plugin.py

```diff
@@ -516,34 +521,23 @@ def _merge_users(self, target, source, **kwargs):
     def get_notification_cc_list(self, action, vc_room, event):
         return {principal_from_identifier(vc_room.data['host']).email}
 
+    def _check_meetings(self, target, changes):
```

Comment:
```suggestion
    def _check_meetings(self, target, changes, **kwargs):
```

always accept extra kwargs in signal handlers